tests/integration_test/statistics_test.go

```diff
@@ -60,8 +60,8 @@ func TestStatistics(t *testing.T) {
 				"http.ingress_http.service_control.allowed":                        1,
 			},
 			wantHistograms: map[string][]float64{
-				// For overhead_time, only verify the maximum(0 will be ignored).
-				"http.ingress_http.service_control.overhead_time": {0, 0, 0, 0, 0, 0, 0, 0, 0, 5},
+				// For overhead_time, don't exact comparison since overhead time varies a log in different build mode.
```

Comment:
the latency is different each run.  Here the backend server introduces a fixed big 1 second latency, so the overall latency should be around 1 second.  This test compares the latency with expected 1 second with certain error margin.

Reply:
Done.